src/plotting/plotWindow.component.tsx

```diff
@@ -38,6 +50,9 @@ const PlotWindow = (props: PlotWindowProps) => {
     );
   }, [plotTitle, untitledTitle]);
 
+  const { data: records, isLoading: recordsLoading } = useRecords();
```

Comment:
Just a thought for the future - we'll likely need to modify `useRecords` to use the values from the `Hours` and/or `Points` fields if they're filled instead of what's defined in the table view. It might be useful to get some clarification from @mattjSTFC on how exactly the users expect to retrieve the relevant data for plots. I think by default it plots based on what's in the table view, but it fetches all the records (so it ignores the pagination parameters).

Reply:
Yes that's correct, as we currently don't have exact clarification on this, this was how I implemented it for the time being. We'll need to track it in Jira issues once we have the info we need

Reply:
Yeah, I'm happy to leave this as it is in this PR and create a separate task to ensure data is being fetched properly once we know how it needs to work.

Reply:
@louise-davies Can you give me more information on what journey requires more detail?

Reply:
In this wireframe: https://balsamiq.cloud/sq9hezy/p37ch0t/r9154 there's two inputs: `Hours` and `Points`. I presume these are to control data fetching? Looking at eCat, these are a feature of continuous mode which I'm not sure if we're supporting anymore? If so, are these inputs needed?

Otherwise, I presume since there's no controls in the plot config to control how data is fetched - that the data is fetched based on the inputs the user has specified in the table view? e.g. the time range, experiment id etc.

Reply:
Thanks for the details. I am not certain either on the definitive purpose of Hours and Points. Let me roll add this question alongside the Time issue and I'll try and get a response back from the CLF team.